pkg/config/libpodConfig.go

```diff
@@ -246,20 +246,6 @@ func readLibpodConfigFromFile(path string, config *ConfigFromLibpod) (*ConfigFro
 		return nil, fmt.Errorf("unable to decode configuration %v: %v", path, err)
 	}
 
-	// For the sake of backwards compat we need to check if the config fields
-	// with *Set suffix are set in the config.  Note that the storage-related
-	// fields are NOT set in the config here but in the storage.conf OR directly
-	// by the user.
-	if config.VolumePath != "" {
-		config.VolumePathSet = true
```

Comment:
Should delete these `VolumePathSet`, `StaticDirSet`, `TmpDirSet`? they are wrapped in the ConfigFromLibpod.SetOptions

Reply:
Good point.